frontend/ui/widget/virtualkeyboard.lua

```diff
@@ -377,6 +377,7 @@ function VirtualKeyPopup:init()
                 virtual_key.onHoldReleaseKey = function()
                     virtual_key:onTapSelect(true)
                     UIManager:close(self)
+                    return true
```

Comment:
Whoops, I didn't mean to commit this line but that's okay. This solves the worst part of #5555.